pkg/api/handlers/compat/info.go

```diff
@@ -55,53 +55,54 @@ func GetInfo(w http.ResponseWriter, r *http.Request) {
-			LoggingDriver:      "",
-			MemTotal:           infoData.Host.MemTotal,
-			MemoryLimit:        sysInfo.MemoryLimit,
-			NCPU:               goRuntime.NumCPU(),
-			NEventsListener:    0,
-			NFd:                getFdCount(),
-			NGoroutines:        goRuntime.NumGoroutine(),
-			Name:               infoData.Host.Hostname,
-			NoProxy:            getEnv("no_proxy"),
-			OSType:             goRuntime.GOOS,
-			OSVersion:          infoData.Host.Distribution.Version,
-			OomKillDisable:     sysInfo.OomKillDisable,
-			OperatingSystem:    infoData.Host.Distribution.Distribution,
-			PidsLimit:          sysInfo.PidsLimit,
+			Architecture:        goRuntime.GOARCH,
+			BridgeNfIP6tables:   !sysInfo.BridgeNFCallIP6TablesDisabled,
+			BridgeNfIptables:    !sysInfo.BridgeNFCallIPTablesDisabled,
+			CPUCfsPeriod:        sysInfo.CPUCfsPeriod,
+			CPUCfsQuota:         sysInfo.CPUCfsQuota,
+			CPUSet:              sysInfo.Cpuset,
+			CPUShares:           sysInfo.CPUShares,
+			CgroupDriver:        configInfo.Engine.CgroupManager,
+			ContainerdCommit:    dockerSystem.Commit{},
+			Containers:          infoData.Store.ContainerStore.Number,
+			ContainersPaused:    stateInfo[define.ContainerStatePaused],
+			ContainersRunning:   stateInfo[define.ContainerStateRunning],
+			ContainersStopped:   stateInfo[define.ContainerStateStopped] + stateInfo[define.ContainerStateExited],
+			Debug:               log.IsLevelEnabled(log.DebugLevel),
+			DefaultAddressPools: getDefaultAddressPools(configInfo),
+			DefaultRuntime:      configInfo.Engine.OCIRuntime,
+			DockerRootDir:       infoData.Store.GraphRoot,
+			Driver:              infoData.Store.GraphDriverName,
+			DriverStatus:        getGraphStatus(infoData.Store.GraphStatus),
+			ExperimentalBuild:   true,
+			GenericResources:    nil,
+			HTTPProxy:           getEnv("http_proxy"),
+			HTTPSProxy:          getEnv("https_proxy"),
+			ID:                  uuid.New().String(),
+			IPv4Forwarding:      !sysInfo.IPv4ForwardingDisabled,
+			Images:              infoData.Store.ImageStore.Number,
+			IndexServerAddress:  "",
+			InitBinary:          "",
+			InitCommit:          dockerSystem.Commit{},
+			Isolation:           "",
+			KernelMemoryTCP:     false,
+			KernelVersion:       infoData.Host.Kernel,
+			Labels:              nil,
+			LiveRestoreEnabled:  false,
+			LoggingDriver:       "",
+			MemTotal:            infoData.Host.MemTotal,
+			MemoryLimit:         sysInfo.MemoryLimit,
+			NCPU:                goRuntime.NumCPU(),
+			NEventsListener:     0,
+			NFd:                 getFdCount(),
+			NGoroutines:         goRuntime.NumGoroutine(),
+			Name:                infoData.Host.Hostname,
+			NoProxy:             getEnv("no_proxy"),
+			OSType:              goRuntime.GOOS,
+			OSVersion:           infoData.Host.Distribution.Version,
+			OomKillDisable:      sysInfo.OomKillDisable,
+			OperatingSystem:     infoData.Host.Distribution.Distribution,
+			PidsLimit:           sysInfo.PidsLimit,
```

Comment:
Ugh, I hate when GitHub makes a one line change like this look like a multi-line touch up.  I think the only real change is the addition of line 72?

			DefaultAddressPools: getDefaultAddressPools(configInfo),

Reply:
yup, gofmt complained, the line was too long by juuust a space